### ISSUE TITLE
java.lang.NullPointerException crashes on Google play console

### ISSUE THREAD

**Issue by mrarronz** [Instabug/Instabug-React-Native#467]
**Steps to Reproduce the Problem**

This crash happened on our app, many users have encountered this crash.

**Expected Behavior**
Instabug should not crash

**Actual Behavior**
Crash happened more than 60 times at this moment, we have released the app for 20% rollout, more than 14k install events.

**Instabug integration code**
We have followed the  official document to complete the implementation. And it's been integrated for a long time with multiple releases.

**SDK Version**
instabug-reactnative: 8.7.3, Not clear which Android version is in this react native module

**React Native, iOS and Android Versions**
instabug-reactnative: 8.7.3

Device Model
Android phone

The crash log from google play console:
```
java.lang.NullPointerException: 
  at com.instabug.library.PresentationManager.show (PresentationManager.java:40)
  at com.instabug.library.b.b (InstabugDelegate.java:496)
  at com.instabug.library.b.a (InstabugDelegate.java:403)
  at com.instabug.library.b$7.run (InstabugDelegate.java:344)
  at android.os.Handler.handleCallback (Handler.java:790)
  at android.os.Handler.dispatchMessage (Handler.java:99)
  at android.os.Looper.loop (Looper.java:164)
  at android.app.ActivityThread.main (ActivityThread.java:7000)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:441)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1408)
```

How to resolve this problem??

**Comment by mrarronz** [comment]
I've checked this crash on https://github.com/Instabug/Instabug-Android/issues/249, and then I upgraded the Instabug-reactnative to v9.0.7. Not sure if this crash is resolved because we cannot reproduce it. 

And a new problem occurred when I use the record screen function of Instabug. The record video button is not disappeared after video is recorded, it always stays on the top of my app screen, the capture image button does the same behavior.

**Comment by alyezz** [comment]
@mrarronz you are correct the crash is fixed on the latest version 9.0.7.

Regarding the recording/capturing button issue, we are unable to reproduce it. Do you have specific reproduction steps or a sample project you could share with us?

**Comment by mrarronz** [comment]
@alyezz Thank you for your quick response, Here is my screenshot for the button issue. `Contact App Support` is the entry to use Instabug related functionalities

![Instabug_issue](https://user-images.githubusercontent.com/10354834/76310169-4283a280-6309-11ea-936b-f7fdd984031a.jpeg)

**Comment by mrarronz** [comment]
instabug-reactnative: v9.0.6/v9.0.7, I've tried both.
**reproduce steps:**

1. Enter the Contact App Support screen
2. Click record video
3. The recording button shows on the screen, click the button, begin recording
4. click stop to complete video recording, return to Instabug screen automatically, noticed that the recording button is not disappeared.
5. click close button on the navigation bar at left, discard changes and back to previous screen, the recording button is there, click it no response

**Comment by mrarronz** [comment]
My initialization code for Instabug component in react-native:
```
Instabug.startWithToken(TOKEN, [Instabug.invocationEvent.shake]);
    Instabug.setWelcomeMessageMode(Instabug.welcomeMessageMode.live);
    Instabug.setViewHierarchyEnabled(true);
    Instabug.setTrackUserSteps(true);
    Instabug.setReproStepsMode(Instabug.reproStepsMode.enabledWithNoScreenshots);
    BugReporting.setShakingThresholdForAndroid(shakeThresholdForAndroid);
    BugReporting.setInvocationOptions([
      BugReporting.invocationOptions.emailFieldOptional,
      BugReporting.invocationOptions.commentFieldRequired,
    ]);
    BugReporting.setEnabled(true);
    Chats.setEnabled(true);
    Replies.setEnabled(true);
    BugReporting.setReportTypes([
      BugReporting.reportType.bug,
      BugReporting.reportType.feedback,
      BugReporting.reportType.question,
    ]);
    Surveys.setShouldShowWelcomeScreen(true);
```

**Comment by mrarronz** [comment]
Here is the code to enter `Contact App Support` screen:

```
BugReporting.show(BugReporting.reportType.bug, [
        BugReporting.option.emailFieldOptional,
        BugReporting.option.commentFieldRequired,
      ]);
```

**Comment by mrarronz** [comment]
@alyezz  I want to know the lowest version of `instabug-reactnative` which has fixed the crash. v9.0.7 is the latest but it's not the best for me since it has such a strange bug, so I would like to try a different version. Thank you, please help me, this is very urgent problem, 'cause we are considering to make a hot fix for our latest release since this crash will affect more users.

**Comment by mrarronz** [comment]
v9.0.5 has the same problem, tested just now. I guess this is a new problem since v9.0

**Comment by alyezz** [comment]
@mrarronz i create a custom build for you based on 8.7.3 containing the fix for the above crash. `npm install --save https://github.com/Instabug/Instabug-React-Native#customBuild/8.7.3.1`
Let me know if this fixes the issue for you

**Comment by mrarronz** [comment]
@alyezz Thank you! I've tried the custom build version, didn't find the above issue. Recording&Capturing both work fine on our release build.

If this version has fixed the crash, I'll push it to production asap, and I would like to keep this version until the recording/capturing bug disappearing from v9.0 and later versions.

Here are detailed steps to reproduce the bug, maybe you can try it again with v9.0.7:

> 1. enter Instabug screen, click record video
> 2. begin video recording, notice that the stop button turns from gray to active, click it to stop video recording. The video has been recorded with duration for 1 second, even less.
> 3. The recording button is not disappeared every time when I stop recording as soon as the stop button turns to active.

I guess maybe this is caused by the video processing so the recording button not disappear in time. Just for your reference, thank you for your help!

**Comment by mrarronz** [comment]
Will post the result here after publishing a hot fix version and see if crash is resolved.

**Comment by mrarronz** [comment]
We have released with this custom build/8.7.3.1 and do not see this crash now, I'm sure it's fixed.
